lib/Map/Building.py

```diff
@@ -9,20 +9,21 @@ class Building:
         - way: List of nodes that defines the shape of the building.
         - coordinate : the coordinate of the building's centroid
     """
-    def __init__(self,buildingId,way):
+    idCounter = itertools.count().__next__
+    def __init__(self, way):
```

Comment:
Would it be interesting to leave the option of defining the buildingId as a parameter in the constructor, in case we load the building from a file?

```python
def __init__(self, way, buildingId = None):
  if not buildingId:
    ...
```

Reply:
In our meeting, we decided to save only the information necessary to skip the heavy calculation right? If this is the case we only need to save the tuple `building.way.id; roadId; (x, y) coordinate` and not the building itself.

If in the future we decide to load the object, we could add this then.